cptv/reader.py

```diff
@@ -293,22 +291,21 @@ def _decompress_frame(self, current_frame, source, packed_bit_width):
         if packed_bit_width == 8:
             s[1:] = source[4:].astype("b")
         else:
-            source = np.append(source, np.zeros(1))  # protect against overrun
-
-            (lookup_high, lookup_low, lookup_bit) = self._fetch_aux(packed_bit_width)
-
-            s[1:] = (
-                256 * source[lookup_high] + source[lookup_low]
-            )  # fetch nearby 16 bits
-
-            # twos complement and shift down into range
-            mask = (1 << packed_bit_width) - 1
-            max_packed_value = 1 << (packed_bit_width - 1)
-            s[1:] = (
-                ((s[1:] >> lookup_bit) + max_packed_value) & mask
-            ) - max_packed_value
-        # TODO: Fast path for 12bit and 16bit
-
+            delta_i = 1
```

Comment:
So...this is a revert back to before the code that Chris wrote but:
  1.  There were lots of other files that were checked in during that time - do we still need them? 
  2.   Do we now the speed differences of the two branches of code for files that they will read?   Maybe it is worth fixing his code because this is something that will speed up our machine learning processing isn't it?

Reply:
1. Its more reverting a small part of the reading code which is responsible for problems. So i think all the other files are needed.

I would like to fix his code but i dont understand it, and so for now i thought it would be best to put a working version back into master and when we have time fix his code.